requirements-test.txt

```diff
@@ -1,3 +1,12 @@
+# Other stuff
+numpy==1.23.3; python_version > "3.7"
+numpy==1.21.5; python_version <= "3.7"
+scipy==1.9.1; python_version > "3.7"
+scipy==1.7.3; python_version <= "3.7"
+
+# Cython
+Cython==0.29.32
```

Comment:
Why do we need Cython again?
I remember we dropped it in https://github.com/BayesWitnesses/m2cgen/commit/f2c42fc897c942cdb447918332a131fc2dc95cd0.

Reply:
Seems like the same reason as in my other comment WRT LightGBM:
```
        File "lightning/setup.py", line 11, in cythonize_extensions
          raise ModuleNotFoundError(
      ModuleNotFoundError: Please install Cython in order to build a lightning from source.
      [end of output]
 ```
 It's actually falling back to compiling from source on my Mac with the M1 chip.

Reply:
Please let me know whether you think we should keep it or drop it based on my use case.

Reply:
OK, I think we can make this dependency conditional then:
```suggestion
# Cython
Cython==0.29.32; sys_platform == 'darwin' and platform_machine == 'arm64'
```

Reply:
Applied your change, thanks